components/properties/src/bc.rs

```diff
@@ -0,0 +1,71 @@
+// This file is part of ICU4X. For terms of use, please see the file
+// called LICENSE at the top level of the ICU4X source tree
+// (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+use crate::BidiDataSource;
+use crate::BidiClass as DataSourceBidiClass
+use icu::properties::{maps, BidiClass};
+use icu_codepointtrie::CodePointTrie;
+
+pub struct BidiClassAdaptor {
+	code_point_trie: CodePointTrie;
+
+}
+
+impl BidiClassAdaptor {
+	pub fn new() -> BidiClassAdaptor {
+		 let provider = icu_testdata::get_provider();
+		 let payload =
+		     maps::get_bidi_class(&provider)
+		         .expect("The data should be valid");
+		 let data_struct = payload.get();
+		 BidiClassAdaptor{ data_struct.code_point_trie }
```

Comment:
```suggestion
		 BidiClassAdaptor{ code_point_trie: data_struct.code_point_trie }
```

Reply:
For now, I added a reference to the provider. However, I would like to have a way to save a reference to the code_point_trie instead. Shall I pass a code_point_trie?

Reply:
I have added it.